### DISCUSSION TITLE
[SUGGESTION] Improvements to Schema Editor

### DISCUSSION THREAD

**Discussion by Parkbyounghyo** [TiagoSilvaPereira/vemto-issues#138] [Ideas & Suggestions]
1. I wish Schema Editor had a table copy function.
2. I hope there is a function to write a field description when working on a table.

**Comment by TiagoSilvaPereira** [comment]
@Parkbyounghyo thank you for your suggestion... could you please share more details?

**Comment by deanlj** [comment]
I'd like to be able to select a table/model in the Schema Editor and copy it and paste it as a new entity, without relationships.

I would be useful to be able to enter a description/note (sql comment?)
![Screenshot 2021-04-29 at 11 01 38](https://user-images.githubusercontent.com/4245429/116534413-640f6700-a8da-11eb-8a0f-0ab191dd4bec.png)
 for a column/field (in the Field Options dialog underneath the Faker field) but copying and pasting entities is more important for my use case.
